SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialWorldSettings.h

```diff
@@ -53,17 +48,16 @@ struct FMapTestingSettings
 UCLASS()
 class SPATIALGDK_API ASpatialWorldSettings : public AWorldSettings
 {
-	GENERATED_BODY()
+	GENERATED_UCLASS_BODY()
 	friend class USpatialStatics;
 
-private:
-	/** Enable running different server worker types to split the simulation. */
-	UPROPERTY(EditAnywhere, Config, Category = "Multi-Worker")
-	bool bEnableMultiWorker;
-
-public:
-	UPROPERTY(EditAnywhere, Category = "Multi-Worker", meta = (EditCondition = "bEnableMultiWorker"))
-	TSubclassOf<USpatialMultiWorkerSettings> MultiWorkerSettingsClass;
+	/** If command line override -OverrideMultiWorkerSettingsClass is set then return the specified class from the command line.
```

Comment:
default access specifier for classes is private, so we need a `public:` block up here

Reply:
I made this property private on purpose so that this can only be accessed by the public function GetMultiWorkerSettingsClass(), which returns the correct case depending on the situation.

Reply:
Ah okay - I was confused by the additional `private` scope below

Reply:
Added the public block.